docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -346,8 +346,8 @@ namespace TeleprompterConsole
 Put that class in a new file, and enclose that class in the
 `TeleprompterConsole` namespace as shown above. You’ll also need to add a `static using`
 statement so that you can reference the `Min` and `Max` method without the
-enclosing class or namespace names. A static using statement imports the
-methods from one class. This is in contrast with the using statements used
+enclosing class or namespace names. A `static using` statement imports the
```

Comment:
If you're going to style the whole thing as inline code then it should be `using static`. Or probably you could also called this static `using` statement.